src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/InternalRequests/NormalRequestTests.cs

```diff
@@ -246,28 +246,6 @@ public async Task NormalRequest_Create_ShouldBeAbleToProposePersonDirectly()
             response.Value.ProposedPerson?.Person.Mail.Should().Be(proposedPerson.Mail);
         }
 
-        [Fact]
-        public async Task NormalRequest_Create_InconcistentDirectAssignment_ShouldGiveBadRequest()
```

Comment:
Guess this was removed since we allow inconsistency between department and person. Or?

Reply:
yep